node/tools/Makefile

```diff
@@ -1,12 +1,34 @@
 .PHONY: run-node node-configs clean
 IP?=127.0.0.1:3054
+DOCKER_IP=172.12.0.10
 
-run-node:
+node:
 	export RUST_LOG=INFO && cd node-configs/${IP} && cargo run -- --database ./database/${IP} 
 
-node-configs:
+node_configs:
 	cargo run --bin localnet_config -- --input-addrs addresses.txt --output-dir node-configs
 
-clean:
+node_docker:
+	rm -rf addresses.txt
+	touch addresses.txt
+	echo ${DOCKER_IP}:3054 >> addresses.txt
+	docker build -t test-node ../
+	docker network create --subnet=172.12.0.0/16 node-net
+	docker run --name test-node-consensus --net node-net --ip ${DOCKER_IP} -itd test-node
+
+consenus_docker_example:
+	rm -rf addresses.txt
+	echo 172.12.0.10:3054 >> addresses.txt
+	echo 172.12.0.11:3054 >> addresses.txt
+	cd .. && docker-compose up -d
+
+clean: clean_docker
 	rm -rf node-configs
 	rm -rf database
+
+clean_docker:
+	docker rm test-consensus-node
+	docker rm node-node-1-1
```

Comment:
AFAIU these names are automatically assigned by docker. Can we make them more stable by specifying them explicitly somewhere?

Reply:
Oh sure, I forgot to change them. Fixed!